### PR TITLE
📝 Update outdated docs urls in metadata files

### PR DESCRIPTION
**Pull request by teallarson**
## What
Changes documentationUrls that pointed to `airbyte.io` to point to `airbyte.com`.

We check for the presence of `airbyte.com` in the documentationUrl to verify that we are showing official documentation before rendering the documentation panel in the webapp.  Now these docs will show again!

## How
That ^

## Recommended reading order
any

## 🚨 User Impact 🚨
Users will see the docs panel for these connectors again

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- Connector version is set to `0.0.1`
    - `Dockerfile` has version `0.0.1`
- Documentation updated
    - Connector's `README.md`
    - Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - `docs/integrations/<source or destination>/<name>.md` including changelog with an entry for the initial version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - `docs/integrations/README.md`

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Connector Generator</strong></summary>

- Issue acceptance criteria met
- PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests#pull-request-title-convention)
- If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

**Review by bnchrch — APPROVED** [review]
Thanks Teal!

**Comment by octavia-squidington-iii** [comment]
##  source-salesforce-singer test report (commit [`efaa126cf1`](https://github.com/airbytehq/airbyte/pull/27420/commits/efaa126cf1fc117164a7831798bfbf4105caac52)) - ❌

⏲️  Total pipeline duration: 29 seconds

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-salesforce-singer/metadata.yaml | ✅        |
| Connector version semver check.                                                 | ✅        |
| QA checks                                                                       | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283064451)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-salesforce-singer test
```

**Comment by tealjulia** [comment]
@bnchrch does this require a version bump as mentioned in the checklist?  I wasn't sure due to it not touching the internal implementation of the connector at all.

**Comment by octavia-squidington-iii** [comment]
##  destination-convex test report (commit [`efaa126cf1`](https://github.com/airbytehq/airbyte/pull/27420/commits/efaa126cf1fc117164a7831798bfbf4105caac52)) - ✅

⏲️  Total pipeline duration: 77 seconds

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-convex/metadata.yaml | ✅        |
| Connector version semver check.                                           | ✅        |
| QA checks                                                                 | ✅        |
| Connector package install                                                 | ✅        |
| Build destination-convex docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                | ✅        |
| Integration tests                                                         | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283064451)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-convex test
```

**Comment by bnchrch** [comment]
@tealjulia Nope! Metadata only changes dont need a version change :)

**Comment by octavia-squidington-iii** [comment]
##  destination-duckdb test report (commit [`efaa126cf1`](https://github.com/airbytehq/airbyte/pull/27420/commits/efaa126cf1fc117164a7831798bfbf4105caac52)) - ❌

⏲️  Total pipeline duration: 97 seconds

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-duckdb/metadata.yaml | ✅        |
| Connector version semver check.                                           | ✅        |
| QA checks                                                                 | ✅        |
| Connector package install                                                 | ✅        |
| Build destination-duckdb docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                | ✅        |
| Integration tests                                                         | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283064451)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-duckdb test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zoom test report (commit [`efaa126cf1`](https://github.com/airbytehq/airbyte/pull/27420/commits/efaa126cf1fc117164a7831798bfbf4105caac52)) - ❌

⏲️  Total pipeline duration: 224 seconds

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zoom/metadata.yaml | ✅        |
| Connector version semver check.                                    | ✅        |
| QA checks                                                          | ✅        |
| Connector package install                                          | ✅        |
| Build source-zoom docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                   | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283064451)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zoom test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-dev-null test report (commit [`efaa126cf1`](https://github.com/airbytehq/airbyte/pull/27420/commits/efaa126cf1fc117164a7831798bfbf4105caac52)) - ❌

⏲️  Total pipeline duration: 442 seconds

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-dev-null/metadata.yaml | ✅        |
| Connector version semver check.                                             | ✅        |
| QA checks                                                                   | ✅        |
| Build connector tar                                                         | ✅        |
| Build destination-dev-null docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ✅        |
| Integration tests                                                           | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283064451)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-dev-null test
```

**Comment by octavia-squidington-iii** [comment]
##  source-e2e-test-cloud test report (commit [`efaa126cf1`](https://github.com/airbytehq/airbyte/pull/27420/commits/efaa126cf1fc117164a7831798bfbf4105caac52)) - ❌

⏲️  Total pipeline duration: 177 seconds

| Step                                                                         | Result   |
|:-----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-e2e-test-cloud/metadata.yaml | ✅        |
| Connector version semver check.                                              | ✅        |
| QA checks                                                                    | ❌        |
| Build connector tar                                                          | ✅        |
| Build source-e2e-test-cloud docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                   | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283064451)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-e2e-test-cloud test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-teradata test report (commit [`efaa126cf1`](https://github.com/airbytehq/airbyte/pull/27420/commits/efaa126cf1fc117164a7831798bfbf4105caac52)) - ❌

⏲️  Total pipeline duration: 543 seconds

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-teradata/metadata.yaml | ✅        |
| Connector version semver check.                                             | ✅        |
| QA checks                                                                   | ✅        |
| Build connector tar                                                         | ✅        |
| Build destination-teradata docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283064451)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-teradata test
```

**Comment by tealjulia** [comment]
@bnchrch is there action I should take on all these failed tests?

**Comment by bnchrch** [comment]
@tealjulia Let me see whats going on

**Comment by octavia-squidington-iii** [comment]
##  source-firebase-realtime-database test report (commit [`924a84ea39`](https://github.com/airbytehq/airbyte/pull/27420/commits/924a84ea392557a9f0c63a573326d55a204a42a2)) - ✅

⏲️  Total pipeline duration: 6 seconds

| Step   | Result   |
|--------|----------|

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283707902)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-firebase-realtime-database test
```

**Comment by octavia-squidington-iii** [comment]
##  source-salesforce-singer test report (commit [`924a84ea39`](https://github.com/airbytehq/airbyte/pull/27420/commits/924a84ea392557a9f0c63a573326d55a204a42a2)) - ❌

⏲️  Total pipeline duration: 20 seconds

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-salesforce-singer/metadata.yaml | ✅        |
| Connector version semver check.                                                 | ✅        |
| QA checks                                                                       | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283707902)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-salesforce-singer test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-convex test report (commit [`924a84ea39`](https://github.com/airbytehq/airbyte/pull/27420/commits/924a84ea392557a9f0c63a573326d55a204a42a2)) - ✅

⏲️  Total pipeline duration: 24 seconds

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-convex/metadata.yaml | ✅        |
| Connector version semver check.                                           | ✅        |
| QA checks                                                                 | ✅        |
| Connector package install                                                 | ✅        |
| Build destination-convex docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                | ✅        |
| Integration tests                                                         | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283707902)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-convex test
```

**Comment by octavia-squidington-iii** [comment]
##  source-braze test report (commit [`924a84ea39`](https://github.com/airbytehq/airbyte/pull/27420/commits/924a84ea392557a9f0c63a573326d55a204a42a2)) - ❌

⏲️  Total pipeline duration: 18 seconds

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-braze/metadata.yaml | ✅        |
| Connector version semver check.                                     | ✅        |
| QA checks                                                           | ✅        |
| Connector package install                                           | ✅        |
| Build source-braze docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                          | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283707902)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-braze test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-duckdb test report (commit [`924a84ea39`](https://github.com/airbytehq/airbyte/pull/27420/commits/924a84ea392557a9f0c63a573326d55a204a42a2)) - ❌

⏲️  Total pipeline duration: 32 seconds

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-duckdb/metadata.yaml | ✅        |
| Connector version semver check.                                           | ✅        |
| QA checks                                                                 | ✅        |
| Connector package install                                                 | ✅        |
| Build destination-duckdb docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                | ✅        |
| Integration tests                                                         | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283707902)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-duckdb test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zoom test report (commit [`924a84ea39`](https://github.com/airbytehq/airbyte/pull/27420/commits/924a84ea392557a9f0c63a573326d55a204a42a2)) - ❌

⏲️  Total pipeline duration: 168 seconds

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zoom/metadata.yaml | ✅        |
| Connector version semver check.                                    | ✅        |
| QA checks                                                          | ✅        |
| Connector package install                                          | ✅        |
| Build source-zoom docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                   | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283707902)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zoom test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-dev-null test report (commit [`924a84ea39`](https://github.com/airbytehq/airbyte/pull/27420/commits/924a84ea392557a9f0c63a573326d55a204a42a2)) - ❌

⏲️  Total pipeline duration: 394 seconds

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-dev-null/metadata.yaml | ✅        |
| Connector version semver check.                                             | ✅        |
| QA checks                                                                   | ✅        |
| Build connector tar                                                         | ✅        |
| Build destination-dev-null docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ✅        |
| Integration tests                                                           | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283707902)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-dev-null test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-teradata test report (commit [`924a84ea39`](https://github.com/airbytehq/airbyte/pull/27420/commits/924a84ea392557a9f0c63a573326d55a204a42a2)) - ❌

⏲️  Total pipeline duration: 540 seconds

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-teradata/metadata.yaml | ✅        |
| Connector version semver check.                                             | ✅        |
| QA checks                                                                   | ✅        |
| Build connector tar                                                         | ✅        |
| Build destination-teradata docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283707902)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-teradata test
```

**Comment by octavia-squidington-iii** [comment]
##  source-e2e-test-cloud test report (commit [`924a84ea39`](https://github.com/airbytehq/airbyte/pull/27420/commits/924a84ea392557a9f0c63a573326d55a204a42a2)) - ❌

⏲️  Total pipeline duration: 177 seconds

| Step                                                                         | Result   |
|:-----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-e2e-test-cloud/metadata.yaml | ✅        |
| Connector version semver check.                                              | ✅        |
| QA checks                                                                    | ❌        |
| Build connector tar                                                          | ✅        |
| Build source-e2e-test-cloud docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                   | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5283707902)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-e2e-test-cloud test
```

**Comment by bnchrch** [comment]
@tealjulia I did some digging

**The tldr is this is a minor change and these are known failures, so lets force merge and ill ensure the metadata files are uploaded**

Below is all the failed connectors, there stage, where they are deployed and whether or not they had been passing our weekly/nightly builds.

They all appear to be alpha, and all but one is currently failing on master. The one that is not is `source-firebase-realtime-database` but after investigating it seems to be a bug in our test complete logic which ive made an issue for (#27425)

Let me know if you agree and we can merge this in :)

| Failed connectors  | Language / Stage | OSS | Cloud | Passing in Master | Status |
| -----------------  | ---------------- | --- | ----- | ----------------- | ------ |
| source-zoom | lowcode/alpha | ✅ | ✅ | ❓ | Failed: Please check your json_schema in selected streams  ('users', 'meetings') |
| source-e2e-test-cloud | java/alpha | ❌ | ✅ | ❓ | unknown |
| destination-teradata | java/alpha | ✅ | ❌ | ❓ | unknown |
| destination-dev-null | java/alpha | ❌ | ✅ | ❓ | unknown |
| destination-duckdb | python/alpha | ✅ | ❌ | ❌ | FAILED integration_tests/integration_test.py::test_write[local_file_config] Invalid Input |
| source-braze | lowcode/alpha | ✅ | ✅ | ❌ | Unit test: TypeError: parse() takes 3 positional arguments but 4 were given |
| source-salesforce-singer | java/alpha | ❌ | ❌ | ❓ | Non issue |
| source-firebase-realtime-database | python/alpha | ✅ | ❌ | ✅ | no such file or directory |

**Comment by tealjulia** [comment]
Sounds good to me @bnchrch
